tests/fullstack-test/expr/hex_str.test

```diff
@@ -0,0 +1,53 @@
+# Copyright 2022 PingCAP, Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+mysql> drop table if exists test.t;
+mysql> create table if not exists test.t(a char(100));
+
+mysql> insert into test.t values('www.pingcap.com');
+mysql> insert into test.t values('abcd');
+mysql> insert into test.t values('测试测试测试测试abcd测试');
+mysql> insert into test.t values(NULL);
+mysql> alter table test.t set tiflash replica 1;
+func> wait_table test t
+
+mysql> set tidb_enforce_mpp=1; set tidb_isolation_read_engines='tiflash'; select hex(a) from test.t;
++----------------------------------------------------------------------+
+| hex(a)                                                               |
++----------------------------------------------------------------------+
+| 7777772e70696e676361702e636f6d                                       |
+| 61626364                                                             |
+| e6b58be8af95e6b58be8af95e6b58be8af95e6b58be8af9561626364e6b58be8af95 |
+| NULL                                                                 |
++----------------------------------------------------------------------+
+
+mysql> drop table if exists test.t;
+mysql> create table if not exists test.t(a varchar(100));
+
+mysql> insert into test.t values('www.pingcap.com');
+mysql> insert into test.t values('abcd');
+mysql> insert into test.t values('测试测试测试测试abcd测试');
+mysql> insert into test.t values(NULL);
+mysql> alter table test.t set tiflash replica 1;
+func> wait_table test t
+
+mysql> set tidb_enforce_mpp=1; set tidb_isolation_read_engines='tiflash'; select hex(a) from test.t;
++----------------------------------------------------------------------+
+| hex(a)                                                               |
++----------------------------------------------------------------------+
+| 7777772e70696e676361702e636f6d                                       |
+| 61626364                                                             |
+| e6b58be8af95e6b58be8af95e6b58be8af95e6b58be8af9561626364e6b58be8af95 |
+| NULL                                                                 |
++----------------------------------------------------------------------+
```

Comment:
Is this a duplicate test case?

Reply:
Oops. I'll replace it with some more meaningful tests.